standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java

```diff
@@ -238,7 +238,9 @@ private void cleanUpZK() {
     }
 
     private void startLocalHDFS() throws IOException {
-        localHdfs = LocalHDFSEmulator.newBuilder().baseDirName("temp").build();
+        String baseDir = "temp";
+        log.info("Starting HDFS Emulator @ {}/", baseDir);
```

Comment:
If this is a directory on disk, can we write an absolute path?

Reply:
The baseDir is used within the context of a temp file so it should be absolute. I will alter the log message to make this clear.